lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/evaluator/EvaluatorManager.java

```diff
@@ -219,6 +219,8 @@ public EvaluatorDescriptor getEvaluatorDescriptor() {
   @Override
   public void close() {
 
+    LOG.log(Level.FINER, "Closing: {0}", this.toString());
```

Comment:
`toString` might be too big. Maybe log `Closing EvaluatorManager for Evaluator "{evaluatorId}"`